tests/cupy_tests/manipulation_tests/test_dims.py

```diff
@@ -298,9 +299,20 @@ def test_broadcast(self, dtype):
         assert broadcast_np.size == broadcast_cp.size
         assert broadcast_np.nd == broadcast_cp.nd
 
+    _broadcast_arrays_hip_skip_condition= [
+        [(1,), (1,)],
+        [(2,), (2,)],
+    ]
+
+    def _check_broadcast_arrays_hip_skip_condition(self):
+        return self.shapes in self._broadcast_arrays_hip_skip_condition
```

Comment:
We can put all together in a function? 😄

```
def _hip_skip_invalid_broadcast(self):
    invalid_shapes = [
        [(1,), (1,)],
        [(2,), (2,)],
    ]
    if runtime.is_hip and self.shapes in invalid_shapes:
        pytest.xfail('HIP/ROCm may have a bug')
```

Reply:
Fixed as such!